domador.js

```diff
@@ -370,12 +371,17 @@ Domador.prototype.process = function process (el) {
     return;
   }
 
+  if (this.inTable) console.log("INTABLE ", el.tagName);
+  if (this.inTable && ['DIV', 'P', 'UL', 'LI', 'PRE', 'BLOCKQUOTE'].indexOf(el.tagName) !== -1) {
```

Comment:
I alphabetized this in the most recent, don't worry. And any other block elements I should add here?